flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalogITCase.java

```diff
@@ -195,4 +195,18 @@ public void testSerialTypes() {
                         + "9223372036854775807]]",
                 results.toString());
     }
+
+    @Test
+    public void testUpper() {
+        List<Row> results =
+                CollectionUtil.iteratorToList(
+                        tEnv.sqlQuery(
+                                        String.format(
+                                                "select * from `%s.%s`",
```

Comment:
May be I missed something but why back ticks here and not double quotes like usually in postgres?

Reply:
Because this sql is Flink SQL not SQL in postgres.